packages/astro/src/@types/astro.ts

```diff
@@ -108,7 +108,6 @@ export interface CLIFlags {
 	drafts?: boolean;
 	open?: boolean;
 	experimentalAssets?: boolean;
-	experimentalMiddleware?: boolean;
```

Comment:
Just commenting that I think there's an entire section of docs to remove from this file! lines 1123 - 1142ish?

Reply:
Thank you @sarah11918 , I completely missed that part!  I updated the PR and removed that section now